linux/ibus-keyman/meson.build

```diff
@@ -0,0 +1,65 @@
+project('ibus-keyman', 'c', 'cpp',
+        version: run_command('cat', '../../VERSION.md', check: true).stdout().strip(),
+        license: 'GPL-2+',
+        # default_options : ['buildtype=release',
+        #                    'cpp_std=c++14',
+        #                    'b_vscrt=static_from_buildtype',
+        #                    'warning_level=2'],
+        meson_version: '>=0.53.0')
+cc = meson.get_compiler('c')
+conf = configuration_data()
+
+ibus =        dependency('ibus-1.0',             version: '>= 1.2.0')
+gtk =         dependency('gtk+-3.0',             version: '>= 2.4')
+x11 =         dependency('x11',                  version: '>= 1.6')
+json_glib =   dependency('json-glib-1.0',        version: '>= 1.0')
+kmnkbp =      dependency('keyman_kmn_processor', version: '>= 0.0.0')
+
+env = find_program('env')
+
+# define some variables. We wouldn't need this if we'd have a meson.build file
+# in the Keyman root directory...
+coredir = '@0@/../../core'.format(meson.current_source_dir())
```

Comment:
The problem with putting a meson.build in the root is it impacts all paths, doesn't it?

That said, I'm not all that excited about linux builds listing all the files in core separately -- it undoes a lot of the good of using meson in the first place. I'd prefer to see us building core as a static library and importing it that way.

Reply:
Good point. Refactored to create a static library and get rid of the direct references of core file.

Reply:
For Core, we need to generate both static library and DLL on Windows -- static library used by Keyman Engine, DLL used by Developer.